libraries/standard/json/src/json.c

```diff
@@ -0,0 +1,1014 @@
+#include <assert.h>
+#include <ctype.h>
+#include <stddef.h>
+#include <stdint.h>
+#include "json.h"
+
+typedef enum bool
```

Comment:
Should we have these guards for C99 /C++ ?
#if defined( __cplusplus ) || ( defined( __STDC_VERSION__ ) && ( __STDC_VERSION__ >= 199901L ) )
    #include <stdbool.h>
#elif !defined( bool )
    typedef enum bool
    {
         true = 1, false = 0
    } bool;
#endif

Reply:
I've changed bool to bool_.  I think I'd like to keep this enum strictly local.